src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/components/SceneFunctionDetailsPanel/components/GitHubContextProvider/infrastructure/GitSessionCookieManager.ts

```diff
@@ -1,6 +1,8 @@
 import { GitSessionCookie } from './GitSessionCookie';
 
 const GITHUB_SESSION_COOKIE_NAME = 'GitSession';
+const NEW_GITHUB_SESSION_COOKIE_NAME = 'pyroscope_git_session';
+const ACCEPTED_GITHUB_SESSION_COOKIE_NAMES = [GITHUB_SESSION_COOKIE_NAME, NEW_GITHUB_SESSION_COOKIE_NAME];
```

Comment:
```suggestion
const GITHUB_SESSION_COOKIE_NAME = 'pyroscope_git_session';
const LEGACY_GITHUB_SESSION_COOKIE_NAME = 'GitSession';
const ACCEPTED_GITHUB_SESSION_COOKIE_NAMES = [GITHUB_SESSION_COOKIE_NAME, LEGACY_GITHUB_SESSION_COOKIE_NAME];
```

I think it makes more sense to change the old name to "legacy."

Reply:
changed